.github/workflows/integrate.yaml

```diff
@@ -28,30 +28,22 @@ jobs:
         uses: "shivammathur/setup-php@v2"
         with:
           php-version: "${{ matrix.php-version }}"
-
-      - name: "Set up Java"
-        uses: "actions/setup-java@v3"
-        with:
-          distribution: "zulu"
-          java-version: "11"
+          tools: "phive"
```

Comment:
Would that be an option for you, @theseer?

Reply:
If it works ;)

And depending on how it it installs it ;)